benchmarks/convert_cifar.py

```diff
@@ -28,6 +36,73 @@ def minimum_bribes(q):
     return bribes
 
 
+def get_size(bytes_count: float, suffix="B"):
+    """
+    Scale bytes to its proper format
+    e.g:
+        1253656 => '1.20MB'
+        1253656678 => '1.17GB'
+    """
+    factor = 1024
+    for unit in ["", "K", "M", "G", "T", "P"]:
+        if bytes_count < factor:
+            return f"{bytes_count:.2f} {unit}{suffix}"
+        bytes_count /= factor
+
+
+def get_system_information():
+    # From https://stackoverflow.com/questions/3103178/how-to-get-the-system-info-with-python
+    info = {}
+    # What is naturally dumped by python-progress-tracker
+    info["ram"] = get_size(psutil.virtual_memory().total)
+    info["cpu"] = cpuinfo.get_cpu_info()["brand_raw"]
+    info["os"] = f"{platform.system()} {platform.release()}"
+
+    # Added metadata about the system
+    info["platform"] = platform.system()
+    info["platform-release"] = platform.release()
+    info["platform-version"] = platform.version()
+    info["architecture"] = platform.machine()
+    info["hostname"] = socket.gethostname()
+    info["processor"] = platform.processor()
+    info["physical_cores"] = psutil.cpu_count(logical=False)
+    info["total_cores"] = psutil.cpu_count(logical=True)
+    uname = platform.uname()
+    info["machine"] = uname.machine
+    info["processor"] = uname.processor
+    info["system"] = uname.system
+    info["node_name"] = uname.node
+    info["release"] = uname.release
+    info["version"] = uname.version
+    info["swap"] = get_size(psutil.swap_memory().total)
+
+    return info
+
+
+def get_ec2_metadata():
+    res = {}
+    try:
+        output = subprocess.check_output("ec2metadata", shell=True, encoding="utf-8")
+        for line in output.split("\n"):
+            if line:
+                splitted = line.split(": ")
+                if len(splitted) == 2:
+                    key, value = splitted
+                    res[key] = value
+            else:
+                print(line)
+        return res
+    except Exception as exception:
+        print(exception)
+        return res
+
+
+def value_else_none(value):
+    if value != value:
+        return None
+    return value
+
```

Comment:
NaNs in the json was failing so we need to make sure that they are None values instead of NaNs

Reply:
could you maybe indicate that in the function's docstring ?